### PR TITLE
Initial file structure

### LINKED ISSUE #2: Start the bot
Add some basic cog structuring, stuff like that

### PR DESCRIPTION
**Pull request by ShakyaMajumdar**
Closes #2.
Replaces #3.

*Edited by dawnofmidnight*

### CODE REVIEW

**Review by dawnofmidnight — APPROVED** [review]
:shipit:

actually, do what Kronifer said

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "bot"
+version = "0.1.0"
+description = ""
+authors = ["cat"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+"discord.py" = "^1.7.1"
```
Comment (Kronifer) [inline pyproject.toml:9]: This needs to be 3.8

File: bot/constants.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+PREFIX = os.getenv("PREFIX") or "!"
+TOKEN = os.getenv("TOKEN")
+
+EXTENSIONS = pathlib.Path("bot/exts/")
+BOT_REPO_URL = "https://github.com/cat-dev-group/bot"
+
+
+class Channels:
+    devlog = 809125747037306960
```
Comment (Kronifer) [inline bot/constants.py:16]: Make this modular by getting from .env

**Review by Kronifer — CHANGES_REQUESTED** [review]
Few nitpicks, maybe stick with a .MD readme

File: bot/constants.py (hunk 2 above)
Comment (dawnofmidnight) [reply bot/constants.py:16]: Not yet. We can pull from yaml later, we should just do this to get started.

Comment (Kronifer) [reply bot/constants.py:16]: Alright resolved

**Review by dawnofmidnight — APPROVED** [review]
Looks good for real this time. We can do proper config stuff when config is set up.

**Review by Kronifer — APPROVED** [review]
LGTM, I'll merge

**Pull request merged**